config/development.json

```diff
@@ -1,5 +1,11 @@
 {
   "logLevel": "debug",
+  "clamdscan": {
```

Comment:
These need switching round. `development.json` will never get run on EC2. We'd want the default settings to contain the standard ubuntu paths, and probably don't need anything in development.json